README.md

```diff
@@ -18,6 +18,28 @@ Example of using a loop :
 {% endfor %}
 ```
 
+### Adding metadata captions
```

Comment:
It would be great to fully generalize this section now by indicating that any metadata can be attached to the image using yaml, and providing the given fields as an example.

I would also place this section after the "files location", at the same heading level.

Ex : 

--------

## Files metadata

You can add any metadata to an image by creating a `.meta.yml` YAML file with the same name next to it.

 ```yml
images/
    bar-image_01.jpg
    bar-image_01.jpg.meta.yml
 ```

You could for example store titles, dates and descriptions to be used as images captions in your theme.

 ```yml
title: Fido Playing with his Bone
year: 2018
description: The other day, Fido got a new bone, and he became really captivated by it.
```
```html
{% for image in images %}
    <img src="{{ image.url }}" alt="{{ image.name }}" {{ image.size }}>
    {% if image.meta %}
        <p>{{ image.meta.title }} ({{ image.meta.year }})<br />{{ image.meta.description }}</p>
    {% endif %}
{% endfor %}
```